src/image/morphology/erode.js

```diff
@@ -101,3 +114,45 @@ function erodeOnceBinary(img, kernel) {
   return newImage;
 }
 
+function erodeOnceBinaryOnlyOnes(img, newImage, kernelWidth, kernelHeight) {
+  const radiusX = (kernelWidth - 1) / 2;
+  const radiusY = (kernelHeight - 1) / 2;
+
+  const minList = [];
+  for (let x = 0; x < img.width; x++) {
+    minList.push(0);
+  }
+
+  for (let y = 0; y < img.height; y++) {
+    let start = y - radiusY;
+    for (let x = 0; x < img.width; x++) {
+      let min = 1;
```

Comment:
Why defining a variable. We can just push 1 in the array and if there is a 0 set minList[]=0 and break;